sky/core.py

```diff
@@ -42,6 +42,8 @@ def status(all: bool, refresh: bool) -> List[Dict[str, Any]]:
                 'last_use': (int) timestamp of last use,
                 'status': (sky.ClusterStatus) cluster status,
                 'autostop': (int) idle time before autostop,
+                'to_down': (bool) whether to tear down the cluster after
```

Comment:
whether autodown is used instead of autostop

Reply:
Done. Thanks!